src/Components/Facility/FacilityHome.tsx

```diff
@@ -594,16 +594,21 @@ export const FacilityHome = (props: any) => {
                 >
                   View Users
                 </DropdownItem>
-                <DropdownItem
-                  id="delete-facility"
-                  variant="danger"
-                  onClick={() => setOpenDeleteDialog(true)}
-                  className="flex items-center gap-3"
-                  icon={<CareIcon className="care-l-trash-alt text-lg" />}
-                  authorizeFor={AuthorizeFor(["DistrictAdmin", "StateAdmin"])}
-                >
-                  Delete Facility
-                </DropdownItem>
+                {authUser.user_type == "DistrictAdmin" ||
+                authUser.user_type == "StateAdmin" ? (
+                  <DropdownItem
+                    id="delete-facility"
+                    variant="danger"
+                    onClick={() => setOpenDeleteDialog(true)}
+                    className="flex items-center gap-3"
+                    icon={<CareIcon className="care-l-trash-alt text-lg" />}
+                    authorizeFor={AuthorizeFor(["DistrictAdmin", "StateAdmin"])}
+                  >
+                    Delete Facility
+                  </DropdownItem>
+                ) : (
+                  <></>
+                )}
```

Comment:
Nice work, but a small change, isn't the authorizeFor redundant now that we are hiding the button in the first place? Authorize for would be a no-op right?

```suggestion
                {authUser.user_type == "DistrictAdmin" ||
                authUser.user_type == "StateAdmin" && (
                  <DropdownItem
                    id="delete-facility"
                    variant="danger"
                    onClick={() => setOpenDeleteDialog(true)}
                    className="flex items-center gap-3"
                    icon={<CareIcon className="care-l-trash-alt text-lg" />}
                  >
                    Delete Facility
                  </DropdownItem>
                )}
```

Reply:
@rithviknishad I just made the suggested changes could you please take a look